tests/introspection/Mac/introspection-mac.csproj

```diff
@@ -26,6 +26,9 @@
     <IncludeMonoRuntime>false</IncludeMonoRuntime>
     <UseSGen>false</UseSGen>
     <PlatformTarget>x86</PlatformTarget>
+    <PackageSigningKey>3rd Party Mac Developer Installer</PackageSigningKey>
```

Comment:
I don't think we should commit this value (`PackageSigningKey`), it can be something else for other people, breaking the build.